src/module/os.c

```diff
@@ -91,3 +85,18 @@ void processCwd(WrenVM* vm)
 
   wrenSetSlotString(vm, 0, buffer);
 }
+
+void processPid(WrenVM* vm) {
+  wrenEnsureSlots(vm, 1);
+  wrenSetSlotDouble(vm, 0, uv_os_getpid());
+}
+
+void processPpid(WrenVM* vm) {
+  wrenEnsureSlots(vm, 1);
```

Comment:
@ruby0x1 Are there any default # of slots I can assume or does EnsureSlots need to ALWAYS be called?  I'm pretty sure I don't see it used everywhere.

Reply:
best to call it each time

Reply:
Is that because we like the explicitness?  It seems in most cases if you just said "there is always a minimum of 2" you'd remove the need for this boilerplate 99% of the time?

Reply:
You can be sure this will be at least as the number of arguments slots. More than that is not guaranteed.

Reply:
Yeah, I'm just asking why, if that is an intentional design decision vs always say having 2 slots which would cover a huge % of use cases.

Reply:
To not require an unnecessary allocation. `wrenEnsureSlots()` is very fast if a grow is not needed.